src/main/java/com/google/javascript/gents/TypeConversionPass.java

```diff
@@ -482,8 +482,11 @@ private void convertTypeAlias() {
       String newTypeName = entry.getValue();
       String filename = typesToFilename.get(oldTypeName);
       FileModule module = fileMap.get(filename);
-      // TypeAnnotationPass will convert the global type name to the local type name using this mapping.
-      module.importedNamespacesToSymbols.put(oldTypeName, newTypeName);
+      if (module != null) {
```

Comment:
not easy to write a test?

Reply:
not hard (fork inner_types test https://github.com/angular/clutz/blob/master/src/test/java/com/google/javascript/gents/singleTests/inner_types.js and remove the first line) but I think the value is limited.